dts/bindings/gpio/gaisler,grgpio.yaml

```diff
@@ -0,0 +1,16 @@
+description: GRLIB GRGPIO - General Purpose I/O Port
```

Comment:
```suggestion
description: Gaisler GRLIB GRGPIO - General Purpose I/O Port
```

Reply:
Updated according to your recommendation.